test/pkg/ptphelper/ptphelper.go

```diff
@@ -666,3 +667,111 @@ func IsExternalGM() (out bool) {
 	logrus.Infof("EXTERNAL_GM=%t", out)
 	return out
 }
+
+func GetListOfWPCEnabledInterfaces(nodeName string) ([]string, string) {
+	var retList = make([]string, 0)
+	var deviceId = ""
+	WPCifaces := getWPCEnabledIfaces(nodeName)
+	for _, iFace := range WPCifaces {
+		if strings.HasSuffix(iFace, "0")  {
+			deviceId, ret := checkGNSSAvailabilityForIface(nodeName, iFace)
+			if (ret) {
+				retList = append(retList, addAllInterfacesForNic(WPCifaces, iFace)...)
+				return retList, deviceId
+			}
+		}
+	}
+	return retList, deviceId
+}
+func addAllInterfacesForNic(WPCifaces map[string]string, firstIface string) []string {
+	var ret = make([]string, 0)
+	for _, iFace := range WPCifaces {
+		if strings.HasPrefix(iFace, strings.TrimSuffix(firstIface, "0")) {
+			ret = append(ret, iFace)
+		}
+	}
+	sort.Slice(ret, func (i,  j int) bool {
+		return ret[i] < ret[j]
+	})
+	return ret
+}
+
+func getWPCEnabledIfaces(nodeName string) map[string]string {
+	resMap := make(map[string]string)
+	cmd := []string{"/bin/sh", "-c", "grep 000e /sys/class/net/*/device/subsystem_device | awk -F '/' '{print $5}'"}
+	so, se, err := execPodCommand(nodeName, cmd)
+	if err != nil {
+		logrus.Errorf("could not get WPC enabled interfaces, err: %s stderr: %s", err, se.String())
+		return resMap
+	}
+	ifaceArr := strings.Split(so.String(), "\n")
+	replacer := strings.NewReplacer("\r", "", "\n", "")
+	for _, iFace := range ifaceArr {
+		if iFace != "" {
+			iFace = replacer.Replace(iFace)
+			resMap[iFace] = iFace
+		}
+	}
+	return resMap
+}
+
+func checkGNSSAvailabilityForIface(nodeName string, IfaceName string) (string, bool) {
+	cmd := []string{"/bin/sh", "-c", fmt.Sprintf("ls /sys/class/net/%s/device/gnss", IfaceName)}
+	logrus.Infof("cmd = %s ", cmd)
+	so, se, err := execPodCommand(nodeName, cmd)
+	if err != nil {
+		logrus.Errorf("could not gnss device, err: %s \n stderr: %s interfaceName %s", err, se.String(), cmd)
```

Comment:
`cmd` is passed to `interfaceName %s`, shouldn't it be `IfaceName`? Also print out `nodeName` to help debug.